packages/core/integration-tests/test/output-formats.js

```diff
@@ -614,7 +614,7 @@ describe('output formats', function() {
       await assertESMExports(b, {a: 2, c: 5});
     });
 
-    it.skip('should support esmodule output (re-export namespace as)', async function() {
+    it.only('should support esmodule output (re-export namespace as)', async function() {
```

Comment:
This tests was skipped previously because it is a known limitation of scope-hoisting.